Maybe/Maybe.csproj

```diff
@@ -11,7 +11,7 @@
     <RepositoryUrl>https://github.com/zbra-solutions/maybe</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Version>0.2.0</Version>
+    <Version>0.3.0</Version>
```

Comment:
Let's update to 1.0.0.

Reply:
Ship it!

Reply:
I believe this should be 0.3.0, we should move to 1.0.0 if we succeed in implementing #36